cratedb_toolkit/io/kinesis/model.py

```
@@ -0,0 +1,45 @@
+    map: t.Union[ColumnTypeDefinition, None] = None
+
+
+@define
+class RecipeDefinition(ProjectTransformation):
+    collections: t.List[CollectionDefinition] = Factory(list)
+    _map: t.Dict[CollectionAddress, CollectionDefinition] = Factory(dict)
+
+    def codec_options(self) -> t.Tuple[PrimaryKeyStore, ColumnTypeMapStore]:
+        pks = PrimaryKeyStore()
+        cms = ColumnTypeMapStore()
+        for collection in self.collections:
```

Pylint (3.3.7)

[error] Non-iterable value self.collections is used in an iterating context

(E1133)